project/plugins.sbt

```diff
@@ -1,14 +1,13 @@
-addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.26")
-addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.4.0")
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.27")
+addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.1")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0-M1")
-addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.3")
+// drop scalafmt on the 1.0.0 branch to dogfood 1.0.0-RC2 before there is a sbt 1.0 of new-sbt-scalafnt
+//  see https://github.com/lucidsoftware/neo-sbt-scalafmt/pull/34
+// addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.3")
 // addSbtPlugin("de.heikoseeberger" % "sbt-header" % "1.7.0")
 addSbtPlugin("org.scala-sbt" % "sbt-houserules" % "0.3.3")
 addSbtPlugin("org.scala-sbt" % "sbt-contraband" % "0.3.0-M9")
-
-resolvers += Resolver.bintrayIvyRepo("scalacenter", "sbt-releases")
-addSbtPlugin(
-  ("com.thesamet" % "sbt-protoc" % "0.99.12")
-    .exclude("com.trueaccord.scalapb", "protoc-bridge_2.12"))
```

Comment:
I remember having the problem you're experiencing. I fixed it by using here `protoc-bridge_2.12` instead of `protoc-bridge_2.10`. I don't know what may have triggered this change again... Try removing the exclusion line.